### DISCUSSION TITLE
Welcome to MerMEId Discussions!

### DISCUSSION THREAD

**Discussion by peterstadler** [Edirom/MerMEId#33] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by tuurma** [comment]
Hello, happy to see the discussions. I work on TEI Publisher, which some of you know has a special component called pb-mei to support musical notation. If some of you would like to share a sample document for Publisher's Demo collection, it would perhaps help people who are not quite sure how to start using MEI in their projects.
